src/PKSim.Core/Snapshots/Services/SnapshotSerializer.cs

```diff
@@ -20,7 +23,12 @@ public class SnapshotSerializer : ISnapshotSerializer
       private static JsonSerializerSettings createSerializerSettings() => new JsonSerializerSettings
       {
          TypeNameHandling = TypeNameHandling.Auto,
-         NullValueHandling = NullValueHandling.Ignore
+         NullValueHandling = NullValueHandling.Ignore,
+         Converters =
+         {
+            new StringEnumConverter()
+         },
+         ContractResolver = new WritablePropertiesOnlyResolver()
```

Comment:
And also I managed to only serialized writable properties => I can use some of the POCO objects from Core